src/System.IO.Packaging/src/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10-beta-23008",
```

Comment:
That's right but we're about to fix this.  What I'd like you to do is the following.
1. Use static versions in the project.json.
2. Do a restore.
3. Modify the project.json to contain *, then modify the lock file to set locked = true, and use *s in the versions in the projectFileDependencyGroups at the end of the file, matching the change in project.json.
4. Confirm that future restores do not modify this file.
This will help ensure that the project is set up correctly so that future updates will happen automatically.